docs/developer-guide/etendo-classic/how-to-guides/how-to-create-a-report-using-process-definition.md

```diff
@@ -25,29 +28,25 @@ https://code.openbravo.com/erp/mods/org.openbravo.platform.features
 
 ### Process Definition
 
-Create a new record in **Process Definition** window. Fields to consider:
+:material-menu: `Application` > `Application Dictionary` > `Process Definition` 
+
+As a `System Administrator` role, create a new record in **Process Definition** window. Fields to consider:
 
 - **UI Pattern**: Set `Report (Using JR Templates)` . This value will display the Report Definition tab.
 
 - **Action Handler**: If none is set when the UI Pattern is selected, the `org.openbravo.client.application.report.BaseReportActionHandler` is set.
 
-![](../../../assets/developer-guide/etendo-classic/how-to-guides/How_to_create_a_Report_using_Process_Definition-1.png)
-
-![](../../../assets/developer-guide/etendo-classic/how-to-guides/How_to_create_a_Report_using_Process_Definition-2.png)
+![](../../../assets/developer-guide/etendo-classic/how-to-guides/how-to-create-a-report-using-process-definition/1.png)
 
 The `BaseReportActionHandler` is the default action handler to use in reports. In case you desire to make some Java validations or include some extra parameters that are not defined in the Process Definition parameters tab it is possible to use a custom Action Handler that extends the `BaseReportActionHandler`.
 
 ### Parameter Definition
 
 In the **Parameter** tab are added all the parameters that are needed to filter the results of the report. Their values are handled by the `BaseReportActionHandler` and sent to Jasper Reports as parameters. These parameters need to be defined in the JR template with the same name as the column name.
 
-![](../../../assets/developer-guide/etendo-classic/how-to-guides/How_to_create_a_Report_using_Process_Definition-3.png)
-
-![](../../../assets/developer-guide/etendo-classic/how-to-guides/How_to_create_a_Report_using_Process_Definition-4.png)
+![](../../../assets/developer-guide/etendo-classic/how-to-guides/how-to-create-a-report-using-process-definition/2.png)
```

Comment:
It is not quite right to show the core as a module (because it cannot be put in development).